index.html

```diff
@@ -38,5 +38,106 @@ <h1>I'm Shimels, Glad to see you!</h1>
         <a href="#"> <img src="icones/Vector.png" alt="" /></a>
       </section>
     </main>
+
+    <main class="project-container">
+      <div class="card">
+        <img src="images/bk-image.jpg" alt="project image" />
+        <h2>Tonic</h2>
+        <h4>
+          <span>CANOPY</span> &nbsp; ● &nbsp; Back End Dev &nbsp; ● &nbsp; 2015
+        </h4>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed euismod
+          euismod nisi, ac bibendum odio bibendum vel. Donec euismod nisl ut
+          magna aliquam, vel lacinia sapien bibendum. Sed eget velit vel velit
+          lacinia dapibus.
+        </p>
+        <ul class="tools">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+
+        <button class="see-project-btn">Show Project</button>
+      </div>
+      <div class="card">
+        <img src="images/bk-image.jpg" alt="project image" />
+        <h2>Tonic</h2>
+        <h4>
+          <span>CANOPY</span> &nbsp; ● &nbsp; Back End Dev &nbsp; ● &nbsp; 2015
+        </h4>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed euismod
+          euismod nisi, ac bibendum odio bibendum vel. Donec euismod nisl ut
+          magna aliquam, vel lacinia sapien bibendum. Sed eget velit vel velit
+          lacinia dapibus.
+        </p>
+        <ul class="tools">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+
+        <button class="see-project-btn">Show Project</button>
+      </div>
+      <div class="card">
+        <img src="images/bk-image.jpg" alt="project image" />
+        <h2>Tonic</h2>
+        <h4>
+          <span>CANOPY</span> &nbsp; ● &nbsp; Back End Dev &nbsp; ● &nbsp; 2015
+        </h4>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed euismod
+          euismod nisi, ac bibendum odio bibendum vel. Donec euismod nisl ut
+          magna aliquam, vel lacinia sapien bibendum. Sed eget velit vel velit
+          lacinia dapibus.
+        </p>
+        <ul class="tools">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+
+        <button class="see-project-btn">Show Project</button>
+      </div>
+    </main>
```

Comment:
- [ ] Nice work implementing the works section :clap: However it would be nice to wrap it with a `section` tag for semantics reasons

- [ ] Kindly make sure sure you stick to the Figma design as much as possible. The `background-color` of your portfolio is different from the Figma design(`#C1C7D0`)

- [ ]  Please add more margin at the top of the "Works" section and more margin between the projects cards to follow the Figma design
### Your design :point_down: 
![Screenshot from 2023-07-07 17-31-04](https://github.com/shimels1/portfolio_microverse/assets/91142432/b3d3d877-81e5-407a-b742-6d28459bc29d)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-06-08 09-34-23](https://github.com/shimels1/portfolio_microverse/assets/91142432/7a7fc166-d528-42b8-ab97-2895d8fc5000)